config/profiles.json

```diff
@@ -7,7 +7,7 @@
     	"tnWeight": 1.0
 	}
   },
-"fp_error": {
+"false_positives": {
```

Comment:
Still not clear. What do you mean by "false_positives"? It doesn't say anything about what is preferred or what is weighted higher or lower.  When looking at results, the profile name should be self explanatory without having to refer back to the profile weights each time.